src/JSONEncoder.py

```diff
@@ -0,0 +1,9 @@
+from eth_utils.conversions import to_hex
+from json import JSONEncoder
+
+class BytesJSONEncoder(JSONEncoder):
+    def default(self, o):
+        if isinstance(o, bytes):
+            return to_hex(o)
+        else:
+            return super(BytesJSONEncoder, self).default(o)
```

Comment:
nit: Newline at EOF

Could you also include a comment here on the reason for this if statement and what scenarios would use which branch of it.
```suggestion
        if isinstance(o, bytes):
            return to_hex(o)
        else:
            return super(BytesJSONEncoder, self).default(o)

```

Reply:
added a comment